src/components/widgets/system/SystemOverviewCard.vue

```diff
@@ -3,6 +3,27 @@
     :title="$t('app.general.title.system_overview')"
     icon="$desktopTower"
   >
+    <template #menu>
+      <v-tooltip
+        bottom
+      >
+        <template #activator="{ on, attrs }">
+          <app-btn
```

Comment:
Eh, I just realized that rolling over the logs restarts the service(s) - so this button should probably be disabled when the printer is busy..

Reply:
Oh, very good point!!

Reply:
Done!